packages/workbox-build/src/entry-points/options/common-generate-schema.js

```diff
@@ -38,16 +38,25 @@ module.exports = baseSchema.keys({
       'staleWhileRevalidate'
     )],
     options: joi.object().keys({
-      cacheName: joi.string(),
-      plugins: joi.array().items(joi.object()),
-      cacheExpiration: joi.object().keys({
-        maxEntries: joi.number().min(1),
-        maxAgeSeconds: joi.number().min(1),
-      }).or('maxEntries', 'maxAgeSeconds'),
+      backgroundSync: joi.object().keys({
+        name: joi.string().required(),
+        options: joi.object(),
+      }),
+      broadcastCacheUpdate: joi.object().keys({
+        channelName: joi.string().required(),
+        options: joi.object(),
```

Comment:
I believe the broadcast cache update object needs a channel name.

Reply:
Yup—that requirement is enforced one line up, on line 45.